templates/provider/hmc/templates/rbac/controller/roles.yaml

```diff
@@ -55,13 +39,6 @@ rules:
   resources:
   - managements
   verbs:
-  - create
-  - delete
-  - get
-  - list
-  - patch
-  - update
-  - watch
```

Comment:
```
{{ include "rbac.editorVerbs" . | nindent 4 }}
```
is missing?

Reply:
no, editorVerbs is wider than needed in this case

Reply:
But we should still allow the controller to get/watch/update/etc. `Management` object. Now, the `verbs` are empty. Or am I missing something?

Reply:
ah, this one is definitely missing, thanks